tests/testsuite/build.rs

```diff
@@ -4179,6 +4179,33 @@ fn uplift_dsym_of_bin_on_mac() {
     assert_that(&p.bin("d.dSYM"), is_not(existing_dir()));
 }
 
+#[test]
+fn uplift_pdb_of_bin_on_windows() {
+    if !cfg!(target_os = "windows") {
```

Comment:
I think this may also want a `target_env` filter

Reply:
Indeed! Updated!